### DISCUSSION TITLE
CentOS make error

### DISCUSSION THREAD

**Discussion by vajrabisj** [vlang/v#15351] [Installation Issues]
make fresh_vc
make[1]: Entering directory `/etc/yum.repos.d/v'
rm -rf ./vc
git clone --depth 1 --quiet --single-branch https://github.com/vlang/vc ./vc
make[1]: Leaving directory `/etc/yum.repos.d/v'
cd ./vc && git clean -xf && git pull --quiet
make fresh_tcc
make[1]: Entering directory `/etc/yum.repos.d/v'
rm -rf ./thirdparty/tcc
git clone --depth 1 --quiet --single-branch --branch thirdparty-linux-amd64 https://github.com/vlang/tccbin ./thirdparty/tcc
fatal: unable to access 'https://github.com/vlang/tccbin/': Encountered end of file
make[1]: *** [fresh_tcc] Error 128
make[1]: Leaving directory `/etc/yum.repos.d/v'
make: *** [thirdparty/tcc/.git/config] Error 2

make failed...

**Comment by JalonSolov** [comment]
Looks more like a connectivity problem...
```
fatal: unable to access 'https://github.com/vlang/tccbin/'
```
